Dockerfile

```diff
@@ -31,11 +31,9 @@ VOLUME [ \
 
 EXPOSE 8050 8051 5023
 
-ENTRYPOINT [ \
-    "python3", \
-    "/app/bin/splash", \
-    "--proxy-profiles-path",  "/etc/splash/proxy-profiles", \
-    "--js-profiles-path", "/etc/splash/js-profiles", \
-    "--filters-path", "/etc/splash/filters", \
-    "--lua-package-path", "/etc/splash/lua_modules/?.lua" \
-]
+# Copy start script
+COPY bin/startup.sh /home/root/startup.sh
+RUN chmod +x /home/root/startup.sh
+
+# Start up script (Generate machine UUID)
+ENTRYPOINT ["/home/root/startup.sh"]
```

Comment:
I think it is not the same as before: previously it was possible to pass command-line arguments to Splash when it is executed from a Docker container (see http://splash.readthedocs.io/en/stable/install.html#passing-custom-options); it looks like you need to handle extra arguments in a script.